cli/waiter/action.py

```diff
@@ -287,3 +287,66 @@ def process_ping_request(clusters, token_name_or_service_id, is_service_id, time
                 success = True
         overall_success = overall_success and success
     return overall_success
+
+
+def send_signal_to_instance_on_cluster(cluster, signal_type, service_id, instance_id, timeout_seconds):
+    """Send sigkill request to the specific instance"""
+    cluster_name = cluster['name']
+    http_util.set_retries(0)
+    try:
+        print(f'Sending {terminal.bold(signal_type)} request to instance {terminal.bold(instance_id)} in {terminal.bold(cluster_name)}...')
+        params = {'timeout': timeout_seconds * 1000, 'instance-id' : instance_id, 'signal-type' : signal_type}
+        resp = http_util.delete(cluster, f'/apps/{service_id}/signal', params=params, read_timeout=timeout_seconds) 
+        logging.debug(f'Response status code: {resp.status_code}')
+        if resp.status_code == 200:
+                    success = resp.json().get("signal-response").get('success')
```

Comment:
Use defensive programming, the json may not be a nested map. Try using `get_in` here to retrieve the value safely.

Reply:
Do you agree that `success = resp.json().get("signal-response", {}).get('success')` accomplishes the same as `get_in`?